implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Test.Services.API.NodeInfoTest do
+  use ExUnit.Case
+
+  alias Ockam.API.Client, as: ApiClient
+  alias Ockam.Services.API.NodeInfo
+
+  test "node info with version" do
+    {:ok, address} = NodeInfo.create(info: %NodeInfo.Info{version: "foo"})
+
+    {:ok, response} = ApiClient.sync_request(:get, "", "", [address])
+
+    assert {:ok, %NodeInfo.Info{version: "foo"}, ""} = NodeInfo.Info.decode(response.body)
+  end
+
+  test "node info without version crashes" do
```

Comment:
```suggestion
  test "node info without version returns error" do
```

Or "does not crash"

Reply:
The process does crash though. `create/1` function returns error. I can validate that it crashes in the test.

Reply:
Oh! That's an unusual design, then, but if we want to uphold that behavior as an important implementation detail, is there any way we can get a pid out somewhere in these the steps and then check for `Process.alive?` at the end?